lib/src/auth/auth.ts

```diff
@@ -1,4 +1,45 @@
-import { AxiosRequestConfig } from 'axios';
+import { JWTPayload, SignJWT } from 'jose';
+
+export type Method =
```

Comment:
nit: I expected to see UPPERCASE here

Reply:
yeah the spec is very shouty, reading it now